pkg/combustion/registry.go

```diff
@@ -9,6 +9,8 @@ import (
 	"slices"
 	"strings"
 
+	"github.com/schollz/progressbar/v3"
+
```

Comment:
Can we fix the import ordering? This empty line and the one below should be removed.